client/src/components/Map.jsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { useState } from "react";
+import ReactMapGL from "react-map-gl";
+import * as Constants from "../Constants";
+
```

Comment:
Actually, I think it's fine to have the constants within this file since they are only going to be used within Map (sorry lol). They should be refactored to CAPTIAL_SNAKE_CASE also.